BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -74,7 +75,27 @@ class ToolbarButton: UIButton, ThemeApplicable {
         configuration = updatedConfiguration
     }
 
-    // MARK: ThemeApplicable
+    private func configureLongPressGestureRecognizerIfNeeded(for element: ToolbarElement) {
+        if element.hasLongPressGestureRecognizer {
```

Comment:
I think it would be better to only add the long press gesture recognizer if the long press action is not nil. So this Bool wouldn't be needed at all.